frame/staking/src/pallet/mod.rs

```diff
@@ -1616,6 +1616,27 @@ pub mod pallet {
 			Self::chill_stash(&stash);
 			Ok(())
 		}
+
+		/// Force a validator to have at least the minimum commission. This will not affect a
+		/// validator who already has a commission greater than or equal to the minimum.
+		#[pallet::weight(T::WeightInfo::force_apply_min_commission())]
+		pub fn force_apply_min_commission(
+			origin: OriginFor<T>,
+			validator_stash: T::AccountId,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+			let min_commission = MinCommission::<T>::get();
+
+			if Validators::<T>::contains_key(&validator_stash) {
+				let _ = Validators::<T>::try_mutate(validator_stash, |prefs| {
```

Comment:
seems like this can be done in one storage call

Reply:
I'll double check, but as far as I can tell this is a `ValueQuery` and the `OnEmpty` impl for this returns the default validator prefs; so if we didn't check that key existed then we would create an entry here because we would get the default, modify it and then write it

Reply:
If the map used an `OptionQuery` we could def skip the `contains_key`

Reply:
There should be a way to backdoor this since in the DB, everything is literally an option and not exposing an option to the runtime developer is just an API thing